versions.sh

```diff
@@ -2,9 +2,9 @@
 
 # Source tarball versions
 
-PG_TARBALL_POSTGRESQL=17beta3
-PG_TARBALL_PGADMIN=8.10
-PG_LP_VERSION=4.3
+PG_TARBALL_POSTGRESQL=17rc1
+PG_TARBALL_PGADMIN=8.11
+PG_LP_VERSION=5.0
```

Comment:
shouldn't it be rc1 installer?

Reply:
Right now the LP installers is created with the name: edb-languagepack-5.0-1-osx.dmg, what should be the name of the installer then?